CMake/cdat_modules_extra/python_configure_step.cmake.in

```diff
@@ -29,7 +29,7 @@ if(APPLE)
 endif()
 
 if(APPLE)
-  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
+  set(library_param --prefix=@CMAKE_INSTALL_PREFIX@ --with-system-expat --enable-framework=@CMAKE_INSTALL_PREFIX@/Library/Frameworks)
```

Comment:
A point to note here is that the [`travis system expat` is `2.0.1`](https://travis-ci.org/PCMDI/pcmdi_metrics/jobs/89462940#L39) whereas the [`Python expat` is `2.1.0`](https://travis-ci.org/PCMDI/pcmdi_metrics/jobs/89462940#L11790)

So querying the `travis` system expat:
```
$ ( echo "from pyexpat import *" ; echo "print EXPAT_VERSION" ) | python
expat_2.0.1
```
And querying the `Python 2.7.10`-built expat (built outside of the `cmake` build system, so I build this prior to the `cmake` commands being called):
```
$ ( echo "from pyexpat import *" ; echo "print EXPAT_VERSION" ) | python
expat_2.1.0
```